index.html

```diff
@@ -295,26 +295,30 @@ <h6 class="title-tag-text">Terminal</h6>
                     so if you'd like to chat please get in touch.
                 </p>
             </div>
-            <form class="contact-form" action="contact">
-                <label for="name" class="fullname">
-                    <input class="name" id="name" type="text" placeholder="Full name">
-                </label>
-                <label for="email" class="email">
-                    <input class="email-address" id="email" type="text" placeholder="Email address">
-                </label>
-                <label for="feedback" class="feedback">
-                    <textarea class="textarea"  name="feedback" placeholder="Enter text here" id="feedback" cols="30" rows="10"></textarea>
-                </label>
+            <form class="contact-form" action="https://formspree.io/f/xqkjaerj" method="post">
+                <ul>
+                    <li class="fullname">
+                        <input class="name" id="name" type="text" placeholder="Full name" maxlength="30" required>
+                    </li>
+                    <li class="email">
+                            <input class="email-address" id="email" type="email" placeholder="Email address" required>
+                    </li>
+                    <li class="feedback">
+                        <textarea class="textarea"  name="feedback" placeholder="Enter text here" id="feedback" cols="30" rows="10" maxlength="500" required></textarea>
+                    </li>
+                </ul>
                <button type="submit" class="get-in-touch">Get in touch</button>
             </form>
```

Comment:
- [ ] Please you should stick to the design as much as possible. Kindly make your colors match the design and also add the background image
*Your design*
![Screenshot 2022-08-23 223110](https://user-images.githubusercontent.com/94551484/186270611-a743ca0f-67b9-41e6-b5d4-c159722faf80.png)

*Figma template*
![Screenshot 2022-08-23 223237](https://user-images.githubusercontent.com/94551484/186270616-ae99064b-3183-4324-a474-3846030a8a2f.png)


- [ ] Please make sure the hover and pressed state on your buttons matches that of the design template.
![Screenshot 2022-08-20 203951](https://user-images.githubusercontent.com/94551484/186270872-ac368b09-777c-4fa6-b551-38ace2e2c700.png)
-